common/app/controllers/EmailSignupController.scala

```diff
@@ -30,59 +30,6 @@ case class EmailForm(
   referrer: Option[String],
   campaignCode: Option[String])
 
-object ListIds {
-  val testList = 3485
-  val guardianTodayUk = 37
-  val guardianTodayUs = 1493
-  val guardianTodayAu = 1506
-
-  val theBestOfOpinion = 2313
-  val newBestOfOpinion = 3811
-  val controlBestOfOpinion = 3814
-  val theFiver = 218
-  val mediaBriefing = 217
-  val brexitBriefing = 3698
-  val greenLight = 38
-  val labNotes = 3701
-  val povertyMatters = 113
-  val theLongRead = 3322
-  val weekendReading = 3743
-  val morningMail = 2636
-  val australianPolitics = 1866
-
-  val theBreakdown = 219
-  val theSpin = 220
-  val guardianAustraliaSports = 3766
-
-  val documentaries = 3745
-  val sleeveNotes = 39
-  val closeUp = 40
-  val filmToday = 1950
-  val bookmarks = 3039
-  val artWeekly = 99
-
-  val zipFile = 1902
-  val theFlyer = 2211
-  val theFlyerCards = 3806
-  val theFlyerConnected = 3807
-  val moneyTalks = 1079
-  val fashionStatement = 105
-  val crosswordEditorUpdate = 101
-  val theObserverFoodMonthly = 248
-
-  val firstDogOnTheMoon = 2635
-  val bestOfOpinionAUS = 2976
-  val bestOfOpinionUS = 3228
-
-  val theGuardianMasterclasses = 3561
-  val theGuardianGardener = 3562
-  val theGuardianBookshop = 3563
-
-  val UsElection = 3599
-
-  val morningMailUk = 3640
-}
-
 object EmailTypes {
```

Comment:
May be outside the scope of this PR, but `EmailTypes` feels like it should go too.

Apparently, only one usage: 

`EmailTypes.landing` is used in `emailLandingBody.scala.html`